### ISSUE TITLE
yagna payment init -r returns successfully (also when run twice), but yagna payment status shows "0" in every row.

### ISSUE THREAD

**Issue by filipgolem** [golemfactory/yagna#681]
```
% yagna payment init -r
% echo $?
0

[2020-10-08T11:24:50Z INFO  ya_gnt_driver::service] init: Init { address: "0x1a4f710a53bcb7a223381efe2ad0d12b3ca79cf5", mode: SEND }
[2020-10-08T11:24:50Z INFO  ya_gnt_driver::gnt::faucet] Requesting Eth from faucet
[2020-10-08T11:24:50Z DEBUG ya_gnt_driver::gnt::faucet] faucet request url: http://51.195.29.1:4000/donate/0x1a4f710a53bcb7a223381efe2ad0d12b3ca79cf5
[2020-10-08T11:24:54Z DEBUG ya_gnt_driver::gnt::faucet] raw faucet response: {"address":"0x1a4f710a53bcb7a223381efe2ad0d12b3ca79cf5","txhash":"0x4bd200f55a6c9450dd22951f4a2a7e622096c30a4e429f5d41921f1e2b2508db","amount":10000000000000000}
[2020-10-08T11:24:54Z INFO  ya_gnt_driver::gnt::faucet] Succesfully requested Eth, waiting...
[2020-10-08T11:25:09Z INFO  ya_gnt_driver::gnt::faucet] Finished requesting Eth from faucet
[2020-10-08T11:25:09Z INFO  ya_gnt_driver] Requesting NGNT from Faucet...
[2020-10-08T11:25:09Z DEBUG ya_gnt_driver::gnt::sender] no reservations pending
[2020-10-08T11:25:12Z INFO  ya_gnt_driver] send new tx: []
[2020-10-08T11:25:12Z INFO  ya_gnt_driver::gnt] Register account: 0x1a4f710a53bcb7a223381efe2ad0d12b3ca79cf5, mode: SEND

[2020-10-08T11:25:14Z DEBUG ya_gnt_driver::gnt::sender] payment job for: "0x1a4f710a53bcb7a223381efe2ad0d12b3ca79cf5"
[2020-10-08T11:25:14Z DEBUG ya_gnt_driver::gnt::sender] Payments: []
[2020-10-08T11:25:14Z DEBUG ya_gnt_driver::gnt::sender] new block: 0xda2a2feeeeebbe6527ee1344d92626cbc46d5a2c76363f9a0d66cdeacdcd4f45
[2020-10-08T11:25:14Z DEBUG ya_gnt_driver::gnt::sender] new block: 0x3de2f62f5257cd662ac0909d9302983a96687358ac17f078349bf039364e79b7

[2020-10-08T11:25:44Z DEBUG ya_gnt_driver::gnt::sender] new block: 0x03c3dd9d530597aa27e553ee2247cde2d8f053a08e5c724953619f350188fc2b
[2020-10-08T11:25:44Z DEBUG ya_gnt_driver::gnt::sender] new block: 0xbb986017fdd2ad6c3ff961023ecd27f348e051ce19adcebb505cfb662a012199
[2020-10-08T11:25:44Z DEBUG ya_gnt_driver::gnt::sender] new block: 0x51611fb67a91f0559b3ee4196ada06f1e3efe071dc541a6f3c6f8a2447de4d3e
[2020-10-08T11:25:44Z DEBUG ya_gnt_driver::gnt::sender] payment job for: "0x1a4f710a53bcb7a223381efe2ad0d12b3ca79cf5"
[2020-10-08T11:25:44Z DEBUG ya_gnt_driver::gnt::sender] Payments: []

% yagna payment status 
---
amount: "0"
incoming:
  accepted: "0"
  confirmed: "0"
  requested: "0"
outgoing:
  accepted: "0"
  confirmed: "0"
  requested: "0"
reserved: "0"
```

**Comment by filipgolem** [comment]
It works now. I think that you can close this issue.

**Comment by maaktweluit** [comment]
This was due to the faucet being slow and not giving out ETH.

We fixed this with #687 to ensure the payment init -r fails when ETH does not arrive in time.
